packages/core/admin/admin/src/content-manager/components/ListViewTable/BulkActionButtons/index.js

```diff
@@ -176,6 +208,137 @@ const ConfirmDialogDeleteAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading
+  React.useEffect(() => {
+    setSelectedEntries(rows.map((row) => row.id));
+  }, [setSelectedEntries, rows]);
+
+  return (
+    <>
+      <Typography>
+        {formatMessage(
+          {
+            id: getTrad('containers.ListPage.selectedEntriesModal.selectedCount'),
+            defaultMessage:
+              '<b>{count}</b> {count, plural, =0 {entries} one {entry} other {entries}} selected',
+          },
+          {
+            count: selectedEntries.length,
+            b: BoldChunk,
+          }
+        )}
+      </Typography>
+      <Box marginTop={5}>
+        <Table.Content>
+          <Table.Head>
+            <Table.HeaderCheckboxCell />
+            <Table.HeaderCell fieldSchemaType="number" label="id" name="id" />
+            {shouldDisplayMainField && (
+              <Table.HeaderCell fieldSchemaType="string" label="name" name="name" />
+            )}
+          </Table.Head>
+          <Tbody>
+            {rows.map((entry, index) => (
+              <Tr key={entry.id}>
+                <Body.CheckboxDataCell rowId={entry.id} index={index} />
+                <Td>
+                  <Typography>{entry.id}</Typography>
+                </Td>
+                {shouldDisplayMainField && (
+                  <Td>
+                    <Typography>{entry[mainField]}</Typography>
+                  </Td>
+                )}
+              </Tr>
+            ))}
+          </Tbody>
+        </Table.Content>
+      </Box>
+    </>
+  );
+};
+
+/* -------------------------------------------------------------------------------------------------
+ * SelectedEntriesModal
+ * -----------------------------------------------------------------------------------------------*/
+
+const SelectedEntriesModal = ({ isOpen, onToggle, onConfirm }) => {
+  const { formatMessage } = useIntl();
+  const { rows, selectedEntries } = useTableContext();
+
+  // Get the selected entries full data, and keep the list view order
+  const entries = rows.filter((row) => {
+    return selectedEntries.includes(row.id);
+  });
+
+  if (!isOpen) {
+    return null;
+  }
+
+  return (
+    <ModalLayout onClose={onToggle} labelledBy="title">
+      <ModalHeader>
+        <Typography fontWeight="bold" textColor="neutral800" as="h2" id="title">
+          {formatMessage({
+            id: getTrad('containers.ListPage.selectedEntriesModal.title'),
+            defaultMessage: 'Publish entries',
+          })}
+        </Typography>
+      </ModalHeader>
+      <ModalBody>
+        <Table.Root rows={entries} colCount={4}>
+          <SelectedEntriesTableContent />
+        </Table.Root>
+      </ModalBody>
+      <ModalFooter
+        startActions={
+          <Button onClick={onToggle} variant="tertiary">
+            {formatMessage({
+              id: 'app.components.Button.cancel',
+              defaultMessage: 'Cancel',
+            })}
+          </Button>
+        }
+        endActions={
+          <Button onClick={onConfirm}>
```

Comment:
Could you go ahead an disable the buttom if there are no selected entries?

Reply:
Currently if you open the modal with selected entries, deselect them all, then click publish it selects everything and publishes. 

We could handle it in another PR too so we can keep moving forward, up to you.

Reply:
Yes I mentioned that in the description, I thought I wouldn't handle it in this PR because it's a bit more complicated and there's a ticket already for everything that happens after clicking "Publish". and if we merge this soon it unlocks the validations part too